intrastat_product/wizards/account_move_reversal.py

```diff
@@ -0,0 +1,18 @@
+# © 2023 FactorLibre - Luis J. Salvatierra <luis.salvatierra@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    def _prepare_default_reversal(self, move):
+        res = super()._prepare_default_reversal(move)
+        res.update(
+            {
+                "intrastat_transport_id": False,
```

Comment:
Explain why these changes, and also add docstring putting this reason/explanation for documenting the code.

Reply:
The change is based on -> https://sede.agenciatributaria.gob.es/Sede/aduanas/intrastat-obligaciones-estadisticas/noticias-interes/nuevos-codigos-naturaleza-transaccion-intrastat.html

On the other hand, I don't understand why code 12 doesn't match with this other documentation... -> https://www.intrastat.com/aeat/normativa/intrastat/tabla-naturaleza-de-la-transaccion-para-intrastat.php

Reply:
Please document this in code.